config/systemd/omnect-ui.env.template

```diff
@@ -5,3 +5,5 @@ LOGIN_USER="%%USER%%"
 LOGIN_PASSWORD="%%PASSWORD%%"
 RUST_LOG="info"
 UI_PORT="1977"
+CERT_PATH=$(echo /var/lib/aziot/certd/certs/deviceid-*.cer),target=/cert/device_id_cert.pem)
+CERT_KEY_PATH=$(echo echo /var/lib/aziot/keyd/keys/deviceid-*.key)
```

Comment:
really double "echo" wanted here?

Reply:
no, I just wanted to save my work (-;
I change to WIP